needrestart

```diff
@@ -1426,33 +1448,25 @@ if($opt_p) {
     exit $ret;
 }
 if ($opt_o) {
-    print "# TYPE needrestart_build info\n";
-    print "# HELP needrestart_build information about needrestart's runtime build\n";
+    print "# TYPE needrestart_build_info gauge\n";
+    print "# HELP needrestart_build_info information about needrestart's runtime build\n";
     print "needrestart_build_info{version=\"$NeedRestart::VERSION\",perl_version=\"$^V\"} 1\n";
 
     if ($opt_k) {
-        my @ometric_kernel_status = map { $_ == $ometric_kernel_values{kresult} ? 1 : 0 } (NRK_NOUPGRADE, NRK_ABIUPGRADE, NRK_VERUPGRADE);
-        print "# TYPE needrestart_kernel_status stateset\n";
+        print "# TYPE needrestart_kernel_status gauge\n";
         print "# HELP needrestart_kernel_status status of kernel as reported by needrestart\n";
-        print "needrestart_kernel_status{needrestart_kernel_status=\"current\"} $ometric_kernel_status[0]\n";
-        print "needrestart_kernel_status{needrestart_kernel_status=\"abi_upgrade\"} $ometric_kernel_status[1]\n";
-        print "needrestart_kernel_status{needrestart_kernel_status=\"version_upgrade\"} $ometric_kernel_status[2]\n";
-	print "# TYPE needrestart_kernel info\n";
+        print "needrestart_kernel_status{needrestart_kernel_status=\"$ometric_kernel_values{kresult}\"} 1\n";
+	print "# TYPE needrestart_kernel gauge\n";
 	print "# HELP needrestart_kernel version information for currenly running and most up to date kernels\n";
-	print "needrestart_kernel_info{running=\"$ometric_kernel_values{krunning}\",expected=\"$ometric_kernel_values{kexpected}\"} 1\n";
+	print "needrestart_kernel{running=\"$ometric_kernel_values{krunning}\",expected=\"$ometric_kernel_values{kexpected}\"} 1\n";
```

Comment:
any reason why you get rid of the `_info` suffix? I understand the *type* `info` is not supported, but it's a well-established convention to name those `_info`, that i would retain here.

Naturally, there's a discordance here between the comment and the actual metric that needs to be fixed regardless, but I would go the other way, if only to harmonize with `needrestart_build_info`...

Reply:
hmm you're right, I should keep those. I'll update just now